.perltidyrc

```diff
@@ -2,8 +2,9 @@
 # # See <https://github.com/houseabsolute/perl-code-tidyall/issues/84>.
 --character-encoding=none
 --no-valign
--l=120   # 120 characters per line
+-l=160
 -fbl     # don't change blank lines
+-fnl     # don't remove new lines
```

Comment:
I agree to include this line but I do not want to loosen the line-length rule. We must find a way to support project specific rule additions and exceptions.

Reply:
As I remember the conversation from the other PR there's no easy way to add exceptions per project. Could 120 be used instead?

Reply:
Okay. I guess I'm fine with 160 then if it's the only deviation.

Reply:
I think we can live with a .perltidyrc per project. If the common project has a config, existing and new projects *can* use this by symlinking, but they don't have to, so openQA would keep is own.
That's at least what @josegomezr agreed on when we chatted today